API.md

```diff
@@ -1421,7 +1421,7 @@ Alternatives can be expressed using the shorter `[]` notation.
 
 ```javascript
 var alt = Joi.alternatives().try(Joi.number(), Joi.string());
-// Same as [Joi.number(), Joi.string()]
+// Same as Joi.alternatives([Joi.number(), Joi.string()])
```

Comment:
How is:

`var alt = Joi.alternatives().try(Joi.number(), Joi.string());`

the same as:

`[Joi.number(), Joi.string()]`


Maybe I'm missing something, but I don't see how the two lines are the same. One's not even being assigned to `alt`